timur/lib/client/scss/table.scss

```diff
@@ -105,72 +105,10 @@
       top: 0;
       z-index: 1;
       /* prevents from scrolling while dragging on touch devices */
-      touch-action :none;
+      touch-action: none;
 
       &.isResizing {
-        background: rgba(forestgreen,0.2);
-      }
-    }
-  }
-}
-
-.tarble-viewer {
```

Comment:
This seems like a typo and isn't used anywhere in the app.

Reply:
lol I do like the typo though.